### PR TITLE
Add thrift_meta etc.

### PR DESCRIPTION
**Pull request by hit9**
@lxyu @maralla

### CODE REVIEW

File: thriftpy/parser/parser.py [hunk 1]
```diff
@@ -454,6 +465,21 @@ def parse(path, module_name=None, include_dirs=None, include_dir=None,
...
+    if not hasattr(thrift, '__thrift_meta__'):
+        meta = {}
+        setattr(thrift, '__thrift_meta__', meta)
+    else:
+        meta = getattr(thrift, '__thrift_meta__')
+
+    if key not in meta:
+        meta[key] = []
+
+    meta[key].append(val)
```
Comment (lxyu) [inline thriftpy/parser/parser.py]: We can use defaultdict here:

```python
if not hasattr(thrift, '__thrift_meta__'):
    setattr(thrift, '__thrift_meta__', collections.defaultdict(list))
thrift.__thrift_meta__.append(val)
```

**Comment by maralla** [comment]
:+1:

File: thriftpy/parser/parser.py (hunk 1 above)
Comment (hit9) [inline thriftpy/parser/parser.py]: :+1:

**Comment by hit9** [comment]
@maralla updated @lxyu

**Pull request merged**